apps/srt-live-transmit.cpp

```diff
@@ -300,6 +300,13 @@ int main( int argc, char** argv )
         return 1;
     }
 
+    if (params[1].rfind("file://con", 0) == 0)
+    {
+        // ensure we are quiet
```

Comment:
That's rather a wrong approach - if a user explicitly requested verbose mode and output to stdout, issue an error and exit with failure. The user should be informed that this combination is impossible, not that it "silently doesn't work".

Reply:
The change will make the application "silently work" when destination is stdout. I think this is way better approach than needlessly barking at the user since user's intention is clear.

Reply:
Users will complain "why the -v option doesn't work?". Moreover, as suggested below, you can make an additional option like `-v2` that will do verbose logs to stderr.